R/TimeStratPetersenNonDiagError_fit.R

```diff
@@ -623,6 +623,7 @@ q <- RunTime(time=time, U=results$sims.list$U, prob=run.prob)
 temp <- rbind(apply(q,2,mean), apply(q,2,sd))
 rownames(temp) <- c("Mean", "Sd")
 print(round(temp,2))
+results$runTime <- (round(temp, 2))
```

Comment:
Not necessary to round the runtiming when saving to the results object as different users may want different rounding.